gemrb/core/Scriptable/Scriptable.cpp

```diff
@@ -1845,9 +1845,7 @@ bool Highlightable::TryUnlock(Actor *actor, bool removekey) const {
 				if (!itemslot)
 					continue;
 				const Item *itemStore = gamedata->GetItem(itemslot->ItemResRef);
-				if (!itemStore)
```

Comment:
Someone has to have this responsibility. Now the code would crash on a null itemStore.